apps/fz_http/lib/fz_http_web/live/rule_live/rule_list_component.ex

```diff
@@ -79,4 +91,12 @@ defmodule FzHttpWeb.RuleLive.RuleListComponent do
   defp user_options(users) do
     Enum.map(users, fn {id, email} -> {email, id} end)
   end
+
+  defp port_type_options do
```

Comment:
Make this a module attribute?

```elixir
@port_type_opts %{TCP: :tcp, UDP: :udp}
```

Reply:
I tried to use a module attribute but it seems that the only way I could then access the attribute from the `heex` is to assign it to the socket, the function seems more tidy than doing that.

Reply:
Kk